### DISCUSSION TITLE
I would like to know how to use my trained model for Perform Speaker Verification.

### DISCUSSION THREAD

**Discussion by kevin00616** [speechbrain/speechbrain#2270] [Q&A] [answered]
### Describe the bug

I attempted to modify the code using the tutorials provided by Hugging Face, but I am unsure how to successfully load my model for verification.
Please help me to solve this problem.

### Expected behaviour

I can successfully perform speaker verification.

**Comment by asumagic** [comment, marked as answer]
I cannot really test myself right now, but I *believe* you should just be able to use the pretrained interfaces, same as pre-trained models on HuggingFace, but specifying a local path instead. Something like:

```python
from speechbrain.pretrained import SpeakerRecognition
verification = SpeakerRecognition.from_hparams(source="./path/to/model", savedir="pretrained_models/spkrec-ecapa-voxceleb")
score, prediction = verification.verify_files("first_test_file.wav", "second_test_file.wav")
```

The exact semantics might depend on the exact SpeechBrain build you based this on as some stuff was changed around how paths are considered in interfaces.

**Reply by kevin00616** [reply]
Thank you for your help. Finally it work.
